src/components/comments-panel.yml

```diff
@@ -1,16 +1,22 @@
 commentsPanel:
-  addCommentButtonText:
-    en: Write comment
-    de: Kommentar schreiben
+  newCommentPlaceholder:
+    en: Enter your comment here
+    de: Gib hier deinen Kommentar ein
   postCommentButtonText:
     en: Post comment
     de: Kommentar abschicken
+  addNewTopicButtonText:
+    en: Add new topic
+    de: Neues Thema hinzufügen
   topicHeaderPrefix:
     en: Topic
     de: Thema
-  newTopicPlaceholder:
+  newTopicHeaderPrefix:
     en: New topic
     de: Neues Thema
+  newTopicPlaceholder:
+    en: e.g. orthography, citation, etc.
```

Comment:
We could write these with capital first letter too, as they are examples of topic names 🤔

Reply:
True. It looked a little bit weird (in both languages actually), but let's stay consistent. Changed it.